modules/gapi/include/opencv2/gapi/infer/ie.hpp

```diff
@@ -351,6 +361,29 @@ template<typename Net> class Params {
         return *this;
     }
 
+    /** @brief Specifies the output precision for model.
+
+    The function is used to set an output precision for model.
+
+    @param precision Precision in OpenCV format.
```

Comment:
Need to explain what is the opencv format.
@alalek Could you explain how to point to opencv types: https://docs.opencv.org/4.x/d1/d1b/group__core__hal__interface.html#ga32b18d904ee2b1731a9416a8eef67d06 in doxygen.

Reply:
`(CV_8U, CV_32F, ...)` ?

Reply:
Done